### PR TITLE
GE: Fix for Batch Processing bug

### PR DESCRIPTION
**Pull request by voduyemi**
Fixed issue with namespaced donation matching rule defaults in non-namespaced orgs

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by voduyemi** [comment]
Link to WI : https://gus.lightning.force.com/a07B0000008EIXlIAO

File: src/lwc/utilCommon/utilCommon.js [hunk 1]
```diff
@@ -514,6 +515,16 @@ const showToast = (title, message, variant, mode, messageData) => {
     dispatchEvent(event);
 }
 
+/*******************************************************************************
+ * @description Strips namespace prefix from object and field api names
+ * @param apiName
+ * @returns {*|string}
+ */
+const stripNameSpace = (apiName) => {
```
[the remark this answers is not in the document]
Comment (psadie-sfdo) [reply src/lwc/utilCommon/utilCommon.js:523]: Additionally would we want to protect against returning `undefined` if the provided apiName doesn't include the namespace prefix?

Maybe something like `if (!apiName.startsWith(namespacePrefix)) return apiName;` or some other guard. Or maybe we _do_ want to return `undefined` for certain scenarios?

**Comment by psadie-sfdo** [comment]
Might need to rename this branch with `feature/...`

File: src/lwc/geBatchWizard/geBatchWizard.js [hunk 2]
```diff
@@ -11,7 +11,12 @@ import {
 } from 'lightning/uiRecordApi';
 import { fireEvent } from 'c/pubsubNoPageRef';
 import { handleError, addKeyToCollectionItems } from 'c/utilTemplateBuilder';
-import { getNestedProperty } from 'c/utilCommon';
+import {
+    getNamespace,
+    getNestedProperty,
+    isNull,
+    stripNameSpace
```
[the remark this answers is not in the document]
Comment (voduyemi) [reply src/lwc/geBatchWizard/geBatchWizard.js:18]: 👍  Bad camel-casing

File: src/lwc/utilCommon/utilCommon.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (voduyemi) [reply src/lwc/utilCommon/utilCommon.js:523]: Absolutely valid suggestions, will make changes.

**Pull request closed without merging**